apps/mull-api/src/app/auth/auth.guard.ts

```diff
@@ -0,0 +1,33 @@
+import { IAuthToken } from '@mull/types';
+import {
+  CanActivate,
+  createParamDecorator,
+  ExecutionContext,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import jwt from 'jsonwebtoken';
+import { Observable } from 'rxjs';
+import { environment } from '../../environments/environment';
+
+@Injectable()
+export class AuthGuard implements CanActivate {
+  canActivate(context: ExecutionContext): boolean | Promise<boolean> | Observable<boolean> {
+    const accessToken = context.switchToHttp().getNext().req.headers.authorization.split(' ')[1];
```

Comment:
I think you should check if the authorization header is present or bring it in the try block since the response returned is a 500 and not a 401 if it's not there. 
![image](https://user-images.githubusercontent.com/36024638/107105866-9b4c0b80-67f6-11eb-86aa-420cc87434eb.png)

Reply:
Good point! Moved it into the try block so we can control what error gets thrown.